lib/controllers/app_controller.dart

```diff
@@ -6,6 +6,13 @@ import 'package:jais/controllers/datas/data_collection_controller.dart';
 import 'package:jais/controllers/url_controller.dart';
 import 'package:jais/firebase_options.dart';
 
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  await Firebase.initializeApp(
+    options: DefaultFirebaseOptions.currentPlatform,
+  );
```

Comment:
As said before, Firebase is initialized in the initialize method of the controller.